README.md

```diff
@@ -1,241 +1,296 @@
+Getting started requires a working environment for using `numba_dpex`. Currently a
+[conda install](#using-a-conda-installation) or a [docker image](#using-the-docker-image)
+are available.
 
 #### Using a conda installation
 
-TODO: update the instructions to install everything from non-dev conda packages
-once available.
-
-##### A conda env with Intel Python libraries (1/2)
-
-Let's create a dedicated conda env with the development versions
-of the Intel Python libraries (from the `dppy/label/dev` channel).
-
-```
-conda create -n dppy-dev dpnp numba cython spirv-tools -c dppy/label/dev -c intel --override-channels
-```
+Conda does not currently support installation of the low-level runtime libraries for
+GPUs, so the first part of the installation guide consists in installing those libraries
+on the host system.
 
-Let's activate it and introspect the available hardware:
+The second part consists in running conda commands that create the environment with
+all the required packages and configuration. Note that the installation logic is a bit
+complicated since it mixes packages from several conda channels `conda-forge`,
+`dppy/label/dev`, and `intel`, some of which being experimental. Neither the builds nor
+the channels are maintained by the `sklearn_numba_dpex` team and their level of
+stability is unknown.
 
-```
-conda activate dppy-dev
-python -c "import dpctl; print(dpctl.get_devices())"
-```
+🚧 TODO: update the instructions to install everything from non-dev conda packages on
+always up-to-date channels whenever it's available.
 
-If you do not not see any CPU device, try again with the following after
-setting the `SYCL_ENABLE_HOST_DEVICE=1` environment variable, for instance:
+##### Install low-level runtime libraries for your GPU (1/2)
 
-```
-SYCL_ENABLE_HOST_DEVICE=1 python -c "import dpctl; print(dpctl.get_devices())"
-```
+At this time, only Intel GPUs are supported.
 
-If you have an Intel GPU and it is not detected, check the
-following steps:
+###### Intel GPU runtime libraries
 
-- Make sure you have installed the [latest GPU drivers](https://dgpu-docs.intel.com/installation-guides/index.html)
+For Intel GPUs, two backends are available. You might want to install both of those,
+and test if one gives better performances.
 
-- On Linux, check that the i915 driver is properly loaded:
+🚧 TODO: write a guide on how to select the device and the backend in a python script.
 
-  ```
-  $ lspci -nnk | grep i915
-        Kernel driver in use: i915
-        Kernel modules: i915
-  ```
+- **Intel Opencl for GPU**: the intel opencl runtime can be installed following
+  [this link](https://github.com/intel/compute-runtime#installation-options).
 
-- On Linux, check that the current user is in the `render` group, for instance:
+  ⚠⚠⚠ **WARNING** ⚠⚠⚠: for Ubuntu (confirmed for `focal` and `jammy`) the `apt`-based
+  installation is broken, see https://github.com/IntelPython/dpctl/issues/887 .
```

Comment:
Get we wrap such technical section within collapsible content (i.e. content wrapped between `<details>` and  `</details>` here)?